mmv1/products/transcoder/JobTemplate.yaml

```diff
@@ -0,0 +1,1137 @@
+                A unique key for this atom.
+              default_from_api: true
+            - !ruby/object:Api::Type::Array
+              name: "inputs"
+              item_type: Api::Type::String
+              description: |
+                List of values identifying files that should be used in this atom.
+              default_from_api: true
+            - !ruby/object:Api::Type::Time
+              name: "startTimeOffset"
+              description: |
+                Start time in seconds for the atom, relative to the input file timeline.
+              default_from_api: true
+            - !ruby/object:Api::Type::Time
+              name: "endTimeOffset"
+              description: |
+                End time in seconds for the atom, relative to the input file timeline.
+              default_from_api: true
+      - !ruby/object:Api::Type::Array
+        name: "elementaryStreams"
+        description: |
+          List of input assets stored in Cloud Storage.
+        default_from_api: true
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+            - !ruby/object:Api::Type::String
+              name: "key"
+              description: |
+                A unique key for this atom.
+              default_from_api: true
+            - !ruby/object:Api::Type::NestedObject
+              name: "videoStream"
+              description: |
+                Encoding of a video stream.
+              default_from_api: true
+              properties:
+                - !ruby/object:Api::Type::NestedObject
+                  name: "h264"
+                  description: |
+                    H264 codec settings
+                  default_from_api: true
+                  properties:
+                    - !ruby/object:Api::Type::Integer
+                      name: "widthPixels"
+                      description: |
+                        The width of the video in pixels.
+                      default_from_api: true
+                    - !ruby/object:Api::Type::Integer
+                      name: "heightPixels"
+                      description: |
+                        The height of the video in pixels.
+                      default_from_api: true
+                    - !ruby/object:Api::Type::Integer
+                      name: "frameRate"
+                      description: The target video frame rate in frames per second (FPS).
+                      required: true
+                    - !ruby/object:Api::Type::Enum
+                      name: "frameRateConversionStrategy"
+                      description: |
+                        Frame rate conversion strategy for desired frame rate.
+                      default_from_api: true
+                      values:
+                        - :FRAME_RATE_CONVERSION_STRATEGY_UNSPECIFIED
+                        - :DOWNSAMPLE
+                        - :DROP_DUPLICATE
+                    - !ruby/object:Api::Type::Integer
```

Comment:
Another example of including more information in the description, from the docs:

> Required. The video bitrate in bits per second. The minimum value is 1,000. The maximum value is 800,000,000